format/format.go

```diff
@@ -133,7 +133,7 @@ func formatValue(value reflect.Value, indentation uint) string {
 	case reflect.Uintptr:
 		return fmt.Sprintf("0x%x", value.Uint())
 	case reflect.Float32, reflect.Float64:
-		return fmt.Sprintf("%v", value.Float())
+		return fmt.Sprintf("%f", value.Float())
```

Comment:
I believe this limits the output to 6 decimal places. Maybe instead use `strconv.FormatFloat` with precision set to -1?
https://golang.org/pkg/strconv/#FormatFloat